index.d.ts

```diff
@@ -0,0 +1,41 @@
+interface Options {
+	/**
+	 * Applies a function to the JSON string before parsing.
+	 */
+	beforeParse?: Function
```

Comment:
I think we can be more specific on what the `Function` is and should return.

```ts
beforeParse?: (data: string) => string;
```

Reply:
Totally agree 😄